applications/nrf5340_audio/src/bluetooth/le_audio_bis_headset.c

```diff
@@ -207,64 +239,73 @@ static void pa_sync_lost_cb(struct bt_audio_broadcast_sink *sink)
 static void base_recv_cb(struct bt_audio_broadcast_sink *sink, const struct bt_audio_base *base)
 {
 	int ret;
-	uint32_t base_bis_index_bitfield = 0U;
-	enum audio_channel channel;
+	bool suitable_stream_found = false;
 
 	if (init_routine_completed) {
 		return;
 	}
 
-	/* Test to ensure there are enough streams for each subgroup */
-	if (base->subgroup_count > CONFIG_BT_AUDIO_BROADCAST_SNK_STREAM_COUNT) {
-		LOG_WRN("Too many channels in subgroup");
-		return;
-	}
-
-	channel_assignment_get(&channel);
-
-	channel = BIT(channel);
+	LOG_DBG("Received BASE with %zu subgroup(s) from broadcast sink", base->subgroup_count);
 
-	LOG_DBG("Received BASE with %d subgroup(s) from broadcast sink", base->subgroup_count);
+	sync_stream_cnt = 0;
```

Comment:
This means that sync_stream_cnt can only be as big as the number of BISes from one source. What if we want one stream from source1 and one from stream2?

Reply:
Will keep as is, can only sync to one source at a time